src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java

```diff
@@ -4870,6 +4870,28 @@ public void testCreateService() throws Exception {
   }
 
   @Test
+  public void testCreateServiceWithWarnings() throws Exception {
```

Comment:
Could you replace this integration test with a unit test instead? See `DefaultDockerClientUnitTest` and https://github.com/spotify/docker-client/blob/master/CONTRIBUTING.md#unit-tests-and-integration-tests